pkg/signup/service/signup_service.go

```diff
@@ -183,8 +183,9 @@ func (s *ServiceImpl) GetSignup(userID string) (*signup.Signup, error) {
 	}
 
 	// Check UserSignup status to determine whether user signup is complete
-	signupCondition, found := condition.FindConditionByType(userSignup.Status.Conditions, v1alpha1.UserSignupComplete)
-	if !found {
+	approvedCondition, approvedFound := condition.FindConditionByType(userSignup.Status.Conditions, v1alpha1.UserSignupApproved)
+	signupCondition, completeFound := condition.FindConditionByType(userSignup.Status.Conditions, v1alpha1.UserSignupComplete)
```

Comment:
to be consistent with the `approvedCondition` variable above, shouldn't we rename `signupCondition` to `completeCondition`?

Reply:
good point, done: 37d3f15